fabfile/db/__init__.py

```diff
@@ -102,9 +103,27 @@ def run(self, action, suffix=None):
         except KeyError:
             print('Error: Unknown engine `%s`' % self.engine)
 
+    # VF: @staticmethod
```

Comment:
We can't use @staticmethod for decorator (postgres fund is a decorator). I can move this func outside the BaseTask class but for consistency I prefered to leave this func inside the class.